api/api.proto

```diff
@@ -391,6 +403,31 @@ message FastpathConfig {
   bool dirty = 5;
 }
 
+// ScanPolicy defines when a filesystem check is triggered and what action to take
+message ScanPolicy {
+    enum ScanTrigger {
+        UnknownScanTrigger = 0;
+        OnMount = 1;
+        OnNextMount = 2;
+        Never = 3;
+    }
+    enum ScanAction {
+        UnknownScanAction = 0;
+        ScanOnly = 1;
+        ScanRepair = 2;
+        None = 3;
+    }
+
+    ScanTrigger Trigger = 1;
+    ScanAction Action = 2;
+}
+
+message ScanPolicies {
+  // ScanPolicy specifies the filesystem check policy to apply for this volume
+  // There can be more than one ScanPolicy associated with a volume
+  repeated ScanPolicy scan_policy = 37;
```

Comment:
this should be 1? i mean repeated ScanPolicy scan_policy = 1; or did you want to reserve some fields?

Reply:
Had a discussion with Veda and Harsh. We will only have one ScanPolicy per volume. ScanPolicies structure would not be required, hence removing it